crypto/cipher_extra/aead_test.cc

```diff
@@ -1364,6 +1364,48 @@ TEST(AEADTest, TestGCMSIV256Change16Alignment) {
   free(encrypt_ctx_256);
 }
 
+TEST(AEADTest, TestMonotonicityCheck) {
+
+  static const uint8_t kEvpAeadCtxKey[32] = {0};
+
+  // Only the tls13() ciphers have monotonicity checks
+  struct {
+    const EVP_AEAD *cipher;
+    const size_t key_len;
+  } ctx[] = { { EVP_aead_aes_128_gcm_tls13(), 16},
+              { EVP_aead_aes_256_gcm_tls13(), 32} };
+
+  for (int i = 0; i < 2; i++) {
+    const EVP_AEAD *cipher = ctx[i].cipher;
+    EVP_AEAD_CTX *encrypt_ctx =
+      (EVP_AEAD_CTX *)malloc(sizeof(EVP_AEAD_CTX) + 8);
```

Comment:
You should use our nice c++ helper classes. The above test is doing weird stuff to test for alignment/ABI isseus and shouldn't be used as a template for "normal" tests. 
```suggestion
    bssl::ScopedEVP_AEAD_CTX encrypt_ctx;
```